tox.ini

```diff
@@ -48,7 +48,7 @@ commands =
     {toxinidir}/run-tests.sh {posargs}
     {toxinidir}/run-func-tests.sh {posargs}
 
-[testenv:py35-postgresql-file-upgrade-from-3.1]
+[testenv:py36-postgresql-file-upgrade-from-3.1]
```

Comment:
Good catch. I needed to change it to test in my env as I don't have Python 3.5 anymore. But I did not check while committing!

Thanks @chungg !